src/tangent_types/thunks.jl

```diff
@@ -1,3 +1,7 @@
+# Disable thunks for 2nd order AD.
+_usethunks() = true
+rrule(::typeof(_usethunks)) = false, (NoTangent(),)
```

Comment:
Shouldn't the second part be a function?
```suggestion
rrule(::typeof(_usethunks)) = false, Returns((NoTangent(),))
```